articles/server-apis/rails.md

```diff
@@ -23,19 +23,16 @@ tags:
 
 **Otherwise, Please follow the steps below to configure your existing Ruby on Rails app to use it with Auth0.**
 
-### 1. Add jwt dependency to your Gemfile
+### 1. Install the knock gem
 
-You need to add the jwt dependency.
-
-Open your Gemfile and add the following:
-
-```bash
-gem 'jwt'
-```
+Setup knock in 2 minutes: 
+<a href="https://github.com/nsarno/knock#getting-started" target="_blank">
```

Comment:
@mgonto I understand!  I'll specify a version number like `gem 'knock', '~> 1.2'` to ensure the documentation stays accurate even if the gem evolves.